src/main/java/cofh/thermaldynamics/duct/fluid/DuctUnitFluid.java

```diff
@@ -367,13 +367,13 @@ public IFluidHandler getFluidCapability(final EnumFacing from) {
 
 				FluidStack info = grid != null ? grid.myTank.getInfo().fluid : null;
 				int capacity = grid != null ? grid.myTank.getInfo().capacity : 0;
-				return new IFluidTankProperties[] { new FluidTankProperties(info, capacity, isOpen(from), isOpen(from)) };
+				return new IFluidTankProperties[] { new FluidTankProperties(info, capacity, isInput(from.ordinal()), isOutput(from.ordinal())) };
 			}
 
 			@Override
 			public int fill(FluidStack resource, boolean doFill) {
 
-				if (isOpen(from) && matchesFilter(from, resource)) {
```

Comment:
context: Servos use `ConnectionType.BLOCKED` and `isOpen()` will always return false if the connectiontype does not allow transfer. We need to check for isInput() instead to control filling the grid from this side.